src/components/MenuPanel/MenuPanelFooter.js

```diff
@@ -13,12 +13,17 @@ const MenuPanelFooter = ({
 }) => {
   return (
     <div css="margin: 15px 20px">
-      <ConnectionBullet connected={connected} />
-      <Text size="xsmall">
-        {connected ? 'Connected to the network' : 'Not connected'}
-      </Text>
-      <ConnectedAccount account={account} onRequestEnable={onRequestEnable} />
-      <PreferencesButton onClick={onOpenPreferences} />
+      <div>
+        <ConnectionBullet connected={connected} />
+        <Text size="xsmall">
+          {connected ? 'Connected to the network' : 'Not connected'}
+        </Text>
+      </div>
+
+      <div css="display: inline-grid; grid-template-rows: 2">
```

Comment:
I'm not sure if `grid-template-rows: 2` is valid syntax; it shows up as invalid in the inspector.

Reply:
Ah yes it’s useless :s